directories/local-conformance-1_2.v2.json

```diff
@@ -23,7 +23,7 @@
       "publisher": "Scott Logic",
       "icons": [
         {
-          "src": "https://www.scottlogic.com/sites/default/files/scott-logic-favicon.jpg"
+          "src": "https://localhost:3001/finos-icon-256.png"
```

Comment:
The logo should be sourced from https://www.scottlogic.com/sites/default/files/scott-logic-favicon.jpg.
Same for all of the other ones in this file